src/app/about/page.tsx

```diff
@@ -1,23 +1,50 @@
 import { Container } from "@/components/shared/container";
 import { Metadata } from "next";
 
+interface SectionProps {
+  title: string;
+  children: React.ReactNode;
+
+}
+
 export const metadata: Metadata = {
   title: "About",
   description: "Learn more about the Legion of Mary",
 };
 
+const Section: React.FC<SectionProps> = ({ title, children }) => (
+  <div className="mb-8">
+    <h2 className="text-2xl font-semibold mb-2">{title}</h2>
+  </div>
+);
+
 const About = () => {
   return (
-    <Container className="">
-      <div className="flex flex-col">
-        <div>About page</div>
-        <div>About local praesidiums</div>
-        <div>About diocese</div>
-        <div>About Curia</div>
-        <div>About Comitium</div>
-        <div>About ???</div>
-      </div>
-    </Container>
+    <>
+       <Container className="max-w-4xl mx-auto p-4 space-y-6">
+        <h1 className="text-4xl font-bold text-center mb-12">The Legion of Mary</h1>
```

Comment:
`mb-8 text-center text-3xl font-bold sm:mb-12 md:text-4xl` can you change to this or something such that it looks gud on mobile

Reply:
Sure!

Reply:
also I inspected and took a look at the mobile view - what exactly did you have in mind for it looking good?